qui/decorators.py

```diff
@@ -290,6 +290,19 @@ def device_domain_hbox(vm, attached: bool) -> Gtk.Box:
 
 
 def create_icon(name) -> Gtk.Image:
-    ''' Create an icon from string '''
-    icon_dev = Gtk.IconTheme.get_default().load_icon(name, 16, 0)
-    return Gtk.Image.new_from_pixbuf(icon_dev)
+    """" Create an icon from string; tries for both the normal and -symbolic
+     variants, because some themes only have the symbolic variant. If not
+     found, outputs a blank icon."""
+
+    names = [name, f'{name}-symbolic']
+    pixbuf = None
+    for icon_name in names:
+        try:
+            pixbuf = Gtk.IconTheme.get_default().load_icon(icon_name, 16, 0)
```

Comment:
If both are present, then it will use -symbolic one. Is that intentional? If not, add `break` here.